app/javascript/components/App.js

```diff
@@ -1,21 +1,18 @@
-import React from 'react';
```

Comment:
The app still does not run due to this error. Kindly see the error.
![error](https://user-images.githubusercontent.com/35504149/185199717-8d2f8239-93d5-49c9-811c-8a2a7f31db77.png)

Reply:
Thank you for reviewing my code, The project work as the project required, on my computer, maybe you can check your yarn, node extra, version on your computer, as a reference, I just linked the record loom video
https://www.loom.com/share/5179525acee64b729197112ee34f95a1